scraper/visualize.py

```diff
@@ -91,7 +92,7 @@ def show_products(products: list[ProductInfo], title: str) -> None:
             product,
             name_format="%website - %name - %id",
         )
-    config_figure(fig, title)
+    config_figure(fig, title) 
     fig.show()
 
```

Comment:
`config={"scrollZoom": True}` is used to enable two finger zooming. Formatted title text also.